.env.example

```diff
@@ -14,4 +14,5 @@ NEXT_PUBLIC_ANON_KEY=
 NEXT_PUBLIC_SUPABASE_URL=
 NEXT_PUBLIC_THIRDWEB_CLIENT_ID=
 PRIVY_APP_ID=
-PRIVY_APP_SECRET=
\ No newline at end of file
+PRIVY_APP_SECRET=
+CHANNEL_FEE=
```

Comment:
out of curiosity, why are you setting this as an ENV variable rather than hard-coding in the `consts` file?